float32 target_blue_val
float32 target_green_val
float32 target_red_val
float32 target_white_val
float32 std_dev_blue
float32 std_dev_green
float32 std_dev_red
float32 std_dev_white
---
uint32 calibrated_exposure
uint16 calibrated_white_balance
float32 final_blue_val
float32 final_green_val
float32 final_red_val
float32 final_white_val
string message
int8 k4aExposureServiceErrorCode